appium/webdriver/extensions/flutter_integration/flutter_finder.py

```diff
@@ -17,11 +17,12 @@
 
 from typing import Tuple
 
-from appium.webdriver.common.appiumby import AppiumBy
+from appium.webdriver.common.appiumby.AppiumBy import By as AppiumBy
+from appium.webdriver.common.appiumby.AppiumBy import ByType
 
 
 class FlutterFinder:
-    def __init__(self, using: str, value: str) -> None:
+    def __init__(self, using: ByType, value: str) -> None:
```

Comment:
Not sure if this is a good idea as it would exclude general selenium locators

Reply:
included